spec/requests/api/v1/statuses_spec.rb

```diff
@@ -126,6 +126,18 @@
           expect(response.headers['X-RateLimit-Limit']).to eq RateLimiter::FAMILIES[:statuses][:limit].to_s
           expect(response.headers['X-RateLimit-Remaining']).to eq (RateLimiter::FAMILIES[:statuses][:limit] - 1).to_s
         end
+
+        it 'returns expected status', :aggregate_failures do
+          subject
+
+          expect(body_as_json).to match a_hash_including(
```

Comment:
Could we add these expectations to that existing example above this one? (change its description to make sense) instead of adding new one? It looks like their requests are the same.

Reply:
check done